tests/virt_autotest/login_console.pm

```diff
@@ -40,14 +40,23 @@ sub login_to_console {
     $timeout //= 5;
     $counter //= 240;
 
+    reset_consoles;
+    select_console 'sol', await_console => 0;
+
```

Comment:
I noticed these two lines moved ahead here, but it will break s390 login.

Reply:
Move reset_consoles after deal s390x

Reply:
looks good.